docs/quick-start.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Quick Start
+description: Learn how to install Apify CLI, and how to create, run, and manage Actors through it.
+sidebar_label: Quick Start
+---
+
+Learn how to install Apify CLI, and how to create, run, and manage Actors through it.
+
+### Step 1: Installation
+
+You can install Apify CLI either using installation scripts:
+
+#### Preferred methods
+
+##### MacOS / Unix
+
+```bash showLineNumbers
```

Comment:
i am not sure if its a good idea to use the line breaks in all those bash examples

Reply:
I removed the `showLineNumbers` from examples, where is just one line